Src/DependencyCollector/NetCore/DependencyCollector.NetCore.csproj

```diff
@@ -19,12 +19,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="..\Shared\DependencyTrackingTelemetryModule.cs;..\Shared\SanitizedHostList.cs;..\Shared\Implementation\DependencyCollectorEventSource.cs;..\Shared\Implementation\ApplicationInsightsUrlFilter.cs;..\Shared\Implementation\RemoteDependencyConstants.cs;..\..\Common\CorrelationIdLookupHelper.cs;..\..\Common\CrossComponentCorrelationEventSource.cs;..\..\Common\ExceptionUtilities.cs;..\..\Common\ICorrelationIdLookupHelper.cs;..\..\Common\RequestResponseHeaders.cs;..\..\Common\HeadersUtilities.cs;..\..\Web\Web.Net45\Implementation\SdkVersionUtils.cs" Exclude="bin\**;obj\**;**\*.csproj;packages\**" />
+    <Compile Include="..\Shared\Implementation\HttpCoreDiagnosticSourceListener.cs;..\Shared\Implementation\HttpHeadersUtilities.cs;..\Shared\Implementation\PropertyFetcher.cs;..\Shared\DependencyTrackingTelemetryModule.cs;..\Shared\SanitizedHostList.cs;..\Shared\Implementation\DependencyCollectorEventSource.cs;..\Shared\Implementation\ApplicationInsightsUrlFilter.cs;..\Shared\Implementation\RemoteDependencyConstants.cs;..\..\Common\CorrelationIdLookupHelper.cs;..\..\Common\CrossComponentCorrelationEventSource.cs;..\..\Common\ExceptionUtilities.cs;..\..\Common\ICorrelationIdLookupHelper.cs;..\..\Common\RequestResponseHeaders.cs;..\..\Common\HeadersUtilities.cs;..\..\Web\Web.Net45\Implementation\SdkVersionUtils.cs" Exclude="bin\**;obj\**;**\*.csproj;packages\**" />
```

Comment:
it is easier to read when every item split in its own `<Compile` node. Also you can include entire `..\Shared` folder instead of individual files. Consider change